<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AzureKinect</class>
 <widget class="QDockWidget" name="AzureKinect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>232</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>0</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
  </property>
  <property name="windowTitle">
   <string>AzureKinect</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Image Format:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_image_format">
           <item>
            <property name="text">
             <string>MJPG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NV12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>YUY2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BGRA32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DEPTH16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IR16</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Color Resolution: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_resolution">
           <item>
            <property name="text">
             <string>OFF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>720P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1080P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1440P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1536P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3072P</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Depth Mode: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_depth_mode">
           <item>
            <property name="text">
             <string>OFF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NFOV_2X2BINNED</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NFOV_UNBINNED</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WFOV_2X2BINNED</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WFOV_UNBINNED</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PASSIVE_IR</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Camera FPS: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_fps">
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">border:2px solid lightgray</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_connect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/connect.svg</normaloff>:/res/icon/connect.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_capture">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Capture</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/capture.svg</normaloff>:/res/icon/capture.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/add.svg</normaloff>:/res/icon/add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reset">
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/reset.svg</normaloff>:/res/icon/reset.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
